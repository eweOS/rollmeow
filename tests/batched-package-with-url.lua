-- Batched packages may have field url set.

return {
	evalDownstream	= function(x) return "11.45.14"; end,
	cachePath	= "/tmp/rollmeow-test.cache.lua",
	packages	= {
		["example"] = {
			url = "https://example.com",
			regex = ".",
			postMatch = function() return "1.2.3"; end,
		};

		["batched-package-with-url"] = {
			url = "https://example.com",
			follow = "example",
		};
	},
};
