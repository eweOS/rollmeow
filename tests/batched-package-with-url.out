batched-package-with-url: upstream 1.2.3 | downstream 11.45.14
example: upstream 1.2.3 | downstream 11.45.14
